modules/cassandra/src/test/java/org/testcontainers/cassandra/CassandraContainerTest.java

```diff
@@ -60,6 +71,46 @@ public void testConfigurationOverride() {
         }
     }
 
+    @Test
+    public void testWithSslClientConfig() {
```

Comment:
let's also include a test using only cqlsh. Similar to https://github.com/testcontainers/testcontainers-java/blob/main/modules/scylladb/src/test/java/org/testcontainers/scylladb/ScyllaDBContainerTest.java#L123-L144

Reply:
Done